DEV_SETUP.md

```diff
@@ -148,7 +137,38 @@ please see [`xmlsec`'s install notes](https://pypi.org/project/xmlsec/).
 
 #### Option A: With `pyenv` and `pyenv-virtualenv`
 
-1. Create the virtualenv `hq` with Python 3.9.11:
+1. Install `pyenv`
+
+  Full installation instructions are [here](https://github.com/pyenv/pyenv#installation)
+  and [here](https://github.com/pyenv/pyenv-installer#installation--update--uninstallation).
+  Check [here](https://github.com/pyenv/pyenv/wiki#troubleshooting--faq)
+  and [here](https://github.com/pyenv/pyenv/wiki/Common-build-problems) to troubleshoot.
+
+  - **Linux**:
+
+    ```sh
+    curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash
+    exec $SHELL
+    ```
+
+  - **macOS**:
+
+    [see installation guide linked above](https://github.com/pyenv/pyenv#installation)
+
+  - Install python 3.9 with `pyenv`:
+
+    ```sh
+    pyenv install 3.9.11
```

Comment:
these should probably be `3.9:latest`

Reply:
nice, I didn't know that was an option - was surprised to see I had to specify a point release with pyenv